projects/risk-assessment-scores-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/messaging/Handler.kt

```diff
@@ -24,7 +24,7 @@ class Handler(
                 "risk-assessment.scores.rsr.determined" -> {
                     riskScoreService.updateRsrScores(
                         message.personReference.findCrn() ?: throw IllegalArgumentException("Missing CRN in ${message.personReference}"),
-                        message.additionalInformation["EventNumber"] as Int,
+                        message.additionalInformation["EventNumber"] as Int? ?: throw DeliusValidationError("No event number provided"),
```

Comment:
I think we'll want to pass this through if it's null, the stored procedure will then infer which event to use.